File: api/Controllers/Models/Entities/Order_Line_Item.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;

namespace IPKP___API.Controllers.Models.Entities
{
    public class Order_Line_Item
    {
        [Key]
        public Guid Order_Line_Item_ID { get; set; }

        [ForeignKey(nameof(Order_Request_ID))]
        public Guid Order_Request_ID { get; set; }

        [ForeignKey(nameof(Personalisation_ID))]
        public Guid Personalisation_ID { get; set; }

        public string Order_Status { get; set; }
        public int Order_Line_Item_Quantity { get; set; }
        public double Order_Line_Item_Total_Price { get; set; }

        //public virtual Order_Request Order_Request { get; set; }
        //public virtual Personalisation_Design Personalisation_Design { get; set; }

        //public virtual ICollection<Order> Order { get; set; }
    }
}

File: admin/src/app/Models/orderlineitem.ts
import { Order_Request } from "./orderrequest";
import { Stock_Item } from "./stockitem";
import { Personalisation_Design } from "./personalisationdesign";

export class Order_Line_Item{
    
    order_Line_Item_ID!: number

    order_Request!: Order_Request
    stock_Item!: Stock_Item
    personalisation_Design!: Personalisation_Design

    stock_Item_Size!: string
    order_Line_Item_Price!: Number
    order_Line_Item_Quantity!: Number
    order_Line_Item_Total_Price!: Number
}